### PR TITLE
make it so people still using email/password (yikes) can actually login

### PR DESCRIPTION
**Pull request by GusCaplan**
:)

### CODE REVIEW

File: src/Client/InternalClient.js [hunk 1]
```diff
@@ -533,6 +533,7 @@ export default class InternalClient {
 	// def login
 	login(email, password) {
 		var client = this.client;
+		client.bot = false;
```
Comment (GusCaplan) [inline src/Client/InternalClient.js]: this was a pretty difficult edit, but i think i managed to pull it off

**Comment by GusCaplan** [comment]
as of 4f7e0a5 this has been verified working

**Comment by abalabahaha** [comment]
Clients using user account can just set the client option bot to true...

**Comment by macdja38** [comment]
What abal probably means by the last comment is that clients using a user account can just set the client option bot to false.

**Comment by GusCaplan** [comment]
@abalabahaha of course i realise that, but we had like 4 people complain about it, so i was like, f**k it, lets just add this in.

**Pull request closed without merging**